src/Concerns/ManagesDataDefinitions.php

```diff
@@ -19,27 +19,32 @@
 
 use Google\Cloud\Core\LongRunning\LongRunningOperation;
 use Google\Cloud\Spanner\Database;
+use function trigger_deprecation;
 
 /**
  * @method Database getSpannerDatabase()
  */
 trait ManagesDataDefinitions
 {
     /**
+     * @deprecated use runDdlBatch() instead
      * @param string $ddl
      * @return LongRunningOperation
      */
     public function runDdl(string $ddl): LongRunningOperation
     {
+        trigger_deprecation('colopl/laravel-spanner', '5.1.1', 'runDdl() is deprecated. Use runDdlBatch() instead.');
```

Comment:
According to CHANGELOG, this change looks like it will be included in v5.2.0.
Which version is the correct one?

Reply:
5.2 is the correct version. this has been fixed. Thanks.